Packs/Orca/Integrations/Orca/Orca.yml

```
@@ -3,10 +3,10 @@ commonfields:
   id: Orca
   version: -1
 configuration:
-- display: API Key
-  name: apikey
+- display: API Token
```

```suggestion
- displaypassword: API Token
  hiddenusername: true
```
and to get the data in the code `demisto.params().get('api_token').get('password')`

---

done